language-server/vscoq-language-server.opam

```diff
@@ -11,7 +11,7 @@ build: [
   [ "dune" "build" "-p" name "-j" jobs ]
 ]
 depends: [
-  "ocaml" { >= "4.13.1" }
+  "ocaml" { >= "4.14" }
```

Comment:
Why the new lower bound ? Do your changes not work with ocaml < 4.14 ?

Reply:
yes, it requires at least 4.14.0 (see https://ocaml.org/p/ppx_yojson_conv/v0.16.0)
and, now it is more consistent in the repo among flake.nix and ci, cd Github actions

Reply:
@gares I think that sounds find ! You can make the final call to merge !